src/module.ts

```diff
@@ -97,20 +97,21 @@ export enum ExpressionId {
   SIMDTernary = 32 /* _BinaryenSIMDTernaryId */,
   SIMDShift = 33 /* _BinaryenSIMDShiftId */,
   SIMDLoad = 34 /* _BinaryenSIMDLoadId */,
-  MemoryInit = 35 /* _BinaryenMemoryInitId */,
-  DataDrop = 36 /* _BinaryenDataDropId */,
-  MemoryCopy = 37 /* _BinaryenMemoryCopyId */,
-  MemoryFill = 38 /* _BinaryenMemoryFillId */,
-  Pop = 39 /* _BinaryenPopId */,
-  RefNull = 40 /* _BinaryenRefNullId */,
-  RefIsNull = 41 /* _BinaryenRefIsNullId */,
-  RefFunc = 42 /* _BinaryenRefFuncId */,
-  Try = 44 /* _BinaryenTryId */,
-  Throw = 45 /* _BinaryenThrowId */,
-  Rethrow = 46 /* _BinaryenRethrowId */,
-  BrOnExn = 47 /* _BinaryenBrOnExnId */,
-  TupleMake = 48 /* _BinaryenTupleMakeId */,
-  TupleExtract = 49 /* _BinaryenTupleExtractId */
+  MemoryInit = 36 /* _BinaryenMemoryInitId */,
```

Comment:
Which one is missing here?

Reply:
I also noticed about all this gaps. Bust can't find any missed parts. Probably it better investigate more but I'm vote fix this issue asap)

Reply:
The missing one appears to be `SIMDLoadStoreLane`: https://github.com/WebAssembly/binaryen/blob/68294338a1cc7337e808671e75933b1134d18a90/src/wasm.h#L573

Reply:
Thanks! Done!

Reply:
Unfortunately c-api missing `_BinaryenSIMDLoadStoreLaneId`. So for now I comment this line